react-components/src/components/Reveal3DResources/Reveal3DResources.tsx

```diff
@@ -136,34 +131,30 @@ export const Reveal3DResources = ({
           />
         );
       })}
-      {styledImage360Collections.image360CollectionAddOptions.map((addModelOption) => {
-        const stylingGroups = styledImage360Collections.image360StyledGroup;
+      {image360CollectionAddOptions.map((addModelOption) => {
         const image360Styling: ImageCollectionModelStyling = {
-          defaultStyle: defaultResourceStyling?.image360?.default ?? new Color(0xff0000),
-          groups: stylingGroups
+          defaultStyle: defaultResourceStyling?.image360?.default,
+          groups: image360StyledGroup
         };
+        let key;
         if ('siteId' in addModelOption) {
-          return (
-            <Image360CollectionContainer
-              key={`${addModelOption.siteId}`}
-              collectionId={addModelOption}
-              styling={image360Styling}
-              onLoad={onModelLoaded}
-              onLoadError={onModelLoadedError}
-            />
-          );
+          key = `${addModelOption.siteId}`;
         } else if ('externalId' in addModelOption) {
+          key = `${addModelOption.externalId}`;
+        }
+
+        if ('externalId' in addModelOption || 'siteId' in addModelOption) {
           return (
             <Image360CollectionContainer
-              key={`${addModelOption.externalId}`}
+              key={key}
               collectionId={addModelOption}
               styling={image360Styling}
               onLoad={onModelLoaded}
               onLoadError={onModelLoadedError}
             />
           );
         }
-        return <></>;
+        return <> </>;
```

Comment:
Not sure if it has consequences, but I'm a little worried about the space here :see_no_evil: